tests/Bridge/Doctrine/MongoDbOdm/PropertyInfo/Fixtures/DoctrineDummy.php

```diff
@@ -81,65 +81,5 @@ class DoctrineDummy
      */
     protected $binUuidRfc4122;
 
-    /**
```

Comment:
Why this removal?

Reply:
PHPStan alert  12 errors that variables are unused